rules.sample.yaml

```diff
@@ -0,0 +1,10 @@
+# A set of regexes to match against joined members in rooms.
```

Comment:
This file should probably have a more descriptive name, like 'provisioning.rules.sample.yaml'.